packages/api/core/test/slow/api_spec_slow.ts

```diff
@@ -78,7 +78,7 @@ for (const nodeInstaller of ['npm', 'yarn']) {
       });
 
       describe('lint', () => {
```

Comment:
Shouldn't this entire block be deleted?

Reply:
Oh never mind, it just lints the rendered templates.

Reply:
I think these tests are still valid in that we want to assert that the code generated from `init` passes our lint command (and we can't `bolt lint` that).

Reply:
jinx lol